lib/proxy/server.rb

```diff
@@ -10,7 +10,7 @@
 #
 class ProxyApp < Sinatra::Base
   enable :logging, :lock
-  set :server, :webrick
+  # set :server, :webrick
```

Comment:
I think you should delete this line (or put it under a flag if that's what's needed), feels like a smell having it commented out